mmpose/datasets/transforms/common_transforms.py

```diff
@@ -286,15 +286,19 @@ class RandomHalfBody(BaseTransform):
     """
 
     def __init__(self,
-                 min_total_keypoints: int = 8,
-                 min_half_keypoints: int = 2,
+                 min_total_keypoints: int = 9,
+                 min_upper_keypoints: int = 2,
+                 min_lower_keypoints: int = 3,
```

Comment:
Is there any reference to set such numbers?

Reply:
The `min_total_keypoints` is corresponding to this: 
https://github.com/open-mmlab/mmpose/blob/183d7dd37e08884d21a1f2e35c74c04cfa96d74f/mmpose/datasets/pipelines/top_down_transform.py#L162


And the `min_upper_keypoints` and `min_lower_keypoints` are corresponding to this:
https://github.com/open-mmlab/mmpose/blob/183d7dd37e08884d21a1f2e35c74c04cfa96d74f/mmpose/datasets/pipelines/top_down_transform.py#L178-L185
according to the selection  preference and internal logic.